### ISSUE TITLE
Overview redesign

### ISSUE THREAD

**Issue by alexlyp** [decred/decrediton#904]
I'd like to begin discussion on what sort of imformation users would like to see displayed on the Overview/Home page in lieu of just the recent transactions.  

I was thinking we could do 1-2 most recent transactions then show some basic staking stats or maybe a news area that would pull in the decred twitter feed or something.  Currently it seems like a lot of wasted space that doesn't provide the user with any additional information

**Comment by andrerfneves** [comment]
I agree there is quite a bit of empty white space in the Home page. I will however say that while using a wallet, I wouldn't expect to see Twitter Feeds embedded (perspective from an end user).

**Comment by lustrousgorilla** [comment]
I like @alexlyp's idea of including some staking stats. What specifically did you have in mind?

I think being able to select an exchange from a dropdown to poll for DCR/BTC and DCR/USD exchange rates would be nice (DCR/USD would obviously be extrapolated from DCR/BTC rate for now). I think a lot of potential users care about the fiat value of their holdings and including it in the wallet would be conducive to a 'Decred-first' mindset, as opposed to scanning coinmarketcap.com and getting distracted seeing the whole landscape of cryptocurrencies.
